fsttest/recorder.go

```diff
@@ -48,3 +49,9 @@ func (r *ResponseRecorder) Close() error {
 // SetManualFramingMode is a no-op on ResponseRecorder.  It exists to
 // satisfy the fsthttp.ResponseWriter interface.
 func (r *ResponseRecorder) SetManualFramingMode(v bool) {}
+
+// Append is a no-op on ResponseRecorder.  It exists to
+// satisfy the fsthttp.ResponseWriter interface.
+func (r *ResponseRecorder) Append(other io.ReadCloser) error {
+	return nil
```

Comment:
This shouldn't be a no-op.  We'll need to copy the bytes into `r.Body` here.

Reply:
Lots of the existing integration tests use `io.Copy` -- it'd be good to update at least one of them to use `Append` instead to exercise this.